domain_model/ElectionRoll.ts

```diff
@@ -6,4 +6,14 @@ export interface ElectionRoll {
     voter_id: Uid; //ID of voter who cast ballot
     ballot_id?:  Uid; //ID of ballot, unsure if this is needed
     submitted: boolean; //has ballot been submitted
-}
\ No newline at end of file
+    state: string;
```

Comment:
Can we go ahead and define the State as an enum?

Reply:
Added in latest commit